indexers/vector/SptagIndexer/__init__.py

```diff
@@ -94,6 +98,10 @@ def build_advanced_index(self, vecs: 'np.ndarray'):
             return _index
 
     def query(self, keys: 'np.ndarray', top_k: int, *args, **kwargs) -> Tuple['np.ndarray', 'np.ndarray']:
+        """Find the top-k vectors with smallest ``metric`` and return their ids in ascending order.
```

Comment:
have u checked if this applies to every metric or if there is an exception? for instance with FaissIndexer the inner product returns similarity, (maybe we should transform to distance ourselves)

Reply:
Good point.
I checked the documentation:
`Vectors returned for a query vector are the vectors that have smallest L2 distance or cosine distances with the query vector.`
Only cosine and l2 are supported.
Also, I added an assertion to the test to make sure the order is ascending.